libs/core/src/lib/table/directives/table-cell.directive.ts

```diff
@@ -8,29 +7,19 @@ import {
     QueryList,
     ContentChildren,
     forwardRef,
-    Optional,
-    HostListener
+    inject,
+    Output,
+    EventEmitter
 } from '@angular/core';
 import { CheckboxComponent } from '@fundamental-ngx/core/checkbox';
-import { DestroyedService, TabbableElementService } from '@fundamental-ngx/cdk/utils';
+import { CellFocusedEventAnnouncer, FocusableItemDirective, FocusableItemPosition } from '@fundamental-ngx/cdk/utils';
 import { Nullable } from '@fundamental-ngx/cdk/utils';
-import { NumberInput } from '@angular/cdk/coercion';
-import { FdTable } from '../fd-table.interface';
-import { TableCellInterface } from '../table-cell.interface';
-
-export interface TableCellPosition {
-    row: number;
-    col: number;
-    totalRows: number;
-    totalCols: number;
-    origin: HTMLTableSectionElement;
-}
 
 @Directive({
     selector: '[fdTableCell], [fd-table-cell]',
-    providers: [DestroyedService, TabbableElementService]
+    hostDirectives: [FocusableItemDirective]
```

Comment:
I think here you could alias the focusable item's `fdkFocusableItem` property to be `focusable`

Reply:
[here is the example](https://angular.io/guide/directive-composition-api#:~:text=Furthermore%2C%20you%20can%20alias%20inputs%20and%20outputs%20from%20hostDirective%20to%20customize%20the%20API%20of%20your%20component%3A)

Reply:
Same goes for output